From 4c13378ace7513de1e7c0688a3fff5fcb79d2186 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Mon, 20 Feb 2017 15:44:40 +0800
Subject: [PATCH 1533/5242] MLK-14007: ASoC: fsl_rpmsg_i2s: remove mono for M4
 don't support it

commit  b21ecc0322071a069ae38e157881067f1cc3a1cf from
https://source.codeaurora.org/external/imx/linux-imx.git

Currently the M4 audio driver don't support mono channel, so remove it.
After mono channel is supported in M4 os, this commit should be reverted.

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/fsl_rpmsg_i2s.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/fsl/fsl_rpmsg_i2s.c b/sound/soc/fsl/fsl_rpmsg_i2s.c
index d8f8da8..10c917e 100644
--- a/sound/soc/fsl/fsl_rpmsg_i2s.c
+++ b/sound/soc/fsl/fsl_rpmsg_i2s.c
@@ -71,14 +71,14 @@ static int i2s_send_message(struct i2s_rpmsg_s *msg,
 static struct snd_soc_dai_driver fsl_rpmsg_i2s_dai = {
 	.playback = {
 		.stream_name = "CPU-Playback",
-		.channels_min = 1,
+		.channels_min = 2,
 		.channels_max = 2,
 		.rates = FSL_RPMSG_I2S_RATES,
 		.formats = FSL_RPMSG_I2S_FORMATS,
 	},
 	.capture = {
 		.stream_name = "CPU-Capture",
-		.channels_min = 1,
+		.channels_min = 2,
 		.channels_max = 2,
 		.rates = FSL_RPMSG_I2S_RATES,
 		.formats = FSL_RPMSG_I2S_FORMATS,
-- 
1.7.9.5

